src/components/Preimage/PreImagesTable.tsx

```
@@ -0,0 +1,174 @@
+									<Address
+										variant='inline'
+										address={preimage.proposer}
+										truncateCharLen={4}
+									/>
+								</div>
+							</TableCell>
+							<TableCell className='font-semibold'>
+								<span className='whitespace-pre font-medium'>{preimage?.deposit && formatBnBalance(preimage?.deposit, { numberAfterComma: 2, withUnit: true }, network)}</span>
+							</TableCell>
+							<TableCell className='font-semibold'>
+								{preimage?.proposedCall && preimage?.proposedCall.section && preimage?.proposedCall.method && (
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)